### ISSUE TITLE
Training enwik8 but loss fail to converge

### ISSUE THREAD

**Issue by KaiPoChang** [lucidrains/compressive-transformer-pytorch#6]
Hi lucidrains,
I appreciate your implementation very much, and it helps me a lot with understanding compressive transformer.
However when I tried running your code (enwik8 and exactly the same code in github), and the loss failed to converge after 100 epochs. Is this in expectation ? Or should I do other additional effort to improve, for example tokenizing the raw data in enwik8 and remove all the xml tags ? The figure below is the training and validation loss while I train enwik8 with the same code as in github.

<img width="426" alt="截圖 2021-03-26 下午5 40 16" src="https://user-images.githubusercontent.com/48987178/112612816-576e9d80-8e5a-11eb-89d7-ac87a03137e0.png">
<img width="410" alt="截圖 2021-03-26 下午5 41 22" src="https://user-images.githubusercontent.com/48987178/112612942-7ec56a80-8e5a-11eb-9ba7-70e020a7cd38.png">

Thanks and look forward to your reply！

**Comment by lucidrains** [comment]
![W B Chart 3_28_2021, 2_03_03 PM](https://user-images.githubusercontent.com/108653/112767950-56f71380-8fce-11eb-8942-fa351e052c17.png)

Hi! Hmm I don't think you could have possibly ran 80 epochs and still have a loss that high. I haven't even hit 1 epoch yet, and I already have a loss under 3

**Comment by lucidrains** [comment]
![W B Chart 3_28_2021, 2_25_27 PM](https://user-images.githubusercontent.com/108653/112768570-73e11600-8fd1-11eb-8594-84ee72c21a4e.png)

generations look good on my end :|
